homeassistant/helpers/aiohttp_client.py

```diff
@@ -111,9 +110,13 @@ def async_cleanup_websession(hass):
     This method is a coroutine.
     """
     tasks = []
-    if DATA_CONNECTOR in hass.data:
+    if DATA_CLIENTSESSION in hass.data:
+        tasks.append(hass.data[DATA_CLIENTSESSION].close())
+    elif DATA_CONNECTOR in hass.data:
```

Comment:
Won't `DATA_CLIENTSESSION` also create a `DATA_CONNECTOR` ([here](https://github.com/pvizeli/home-assistant/blob/33edf322df4b5f76099b30aaeabf93c6aa6307fb/homeassistant/helpers/aiohttp_client.py#L31))? In that case this must be `if` and not `elif`